src/circuits/rtable.rs

```diff
@@ -160,9 +164,12 @@ impl<F: FieldExt> RangeTableConfig<F> {
         &self,
         meta: &mut ConstraintSystem<F>,
         key: &'static str,
-        expr: impl FnOnce(&mut VirtualCells<'_, F>) -> Expression<F>,
+        expr: impl FnOnce(&mut VirtualCells<'_, F>) -> [Expression<F>; 2],
     ) {
-        meta.lookup(key, |meta| vec![(expr(meta), self.pow_col)]);
+        meta.lookup(key, |meta| {
+            let [e0, e1] = expr(meta);
+            vec![(e0, self.pow_col[0]), (e1.clone(), self.pow_col[1])]
```

Comment:
try `pow_table_power_encode(e1.clone())` here to avoid calling in instruction configuration and assignment.

Reply:
We can't call encode on configuration. Because avaiable sets are (0, PREFIX + 0, PREFIX + 1, ...). If we force to add PREFIX in configure, we cannot reach value 0.